src/components/header/v2/secondHeader.jsx

```diff
@@ -108,17 +106,18 @@ const SecondHeader = ({
             tabIndex={-1}
           >
             {tabList.map(item => {
-              const { id, label } = item;
+              let { id, label, href } = item;
+              href = locale === 'en' ? href : `/cn${href}`;
```

Comment:
don't directly change parameters,  use another variable or put the statement in <Link />